### PR TITLE
chore: un-nest CLIs, APIs, permissions and DB layer

### PR DESCRIPTION
**Pull request by ShreyaLnuHpe**
## Ticket

[DET-10462](https://hpe-aiatscale.atlassian.net/browse/DET-10462)
[DET-10465](https://hpe-aiatscale.atlassian.net/browse/DET-10465)

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-10462]: https://hpe-aiatscale.atlassian.net/browse/DET-10462?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-10465]: https://hpe-aiatscale.atlassian.net/browse/DET-10465?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: master/static/migrations/20241010174718_alter-user-session-add-access-token-info.tx.up.sql
Comment (ShreyaLnuHpe) [inline master/static/migrations/20241010174718_alter-user-session-add-access-token-info.tx.up.sql:1]: Ignore the migration files as they were mistakenly added lines and then got pushed to feature branch.

File: harness/determined/cli/token.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    token_store = authentication.TokenStore(args.master)
+    token_store.set_token(username, args.token)
+    token_store.set_active(username)
+    print(f"Authenticated as {username}.")
+
+
+# fmt: off
+
+args_description = [
+    cli.Cmd("to|ken", None, "manage access tokens", [
```
Comment (corban-beaird) [inline harness/determined/cli/token.py:143]: I like that you added a shortened version of token here, adds a clearer polish to this new set of subcommands!

File: master/static/migrations/20241010174718_alter-user-session-add-access-token-info.tx.up.sql
Comment (corban-beaird) [reply master/static/migrations/20241010174718_alter-user-session-add-access-token-info.tx.up.sql:1]: If you're not looking to push these changes here, could you potentially rebase against the feature branch or remove these changes from the commit using something like `git reset`?

**Review by corban-beaird — APPROVED** [review]
Looks solid! Thanks for getting this knocked out so quickly! Maybe just do a quick rebase against the feature branch to clean up the extra migrations commits and this should be good to go!

File: master/static/migrations/20241010174718_alter-user-session-add-access-token-info.tx.up.sql
Comment (ShreyaLnuHpe) [reply master/static/migrations/20241010174718_alter-user-session-add-access-token-info.tx.up.sql:1]: I actually added these DROP commands in the `shreya/createTable` (feature branch), so resolving that by removing these lines in this PR.

**Pull request merged**